### PR TITLE
bump rustfmt to v1.4.32

### PR DESCRIPTION
**Pull request by calebcartwright**
Fixes an indentation bug with bounds reported in https://github.com/rust-lang/rust/pull/80843#discussion_r554476529

r? @Mark-Simulacrum

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -5346,7 +5346,7 @@ dependencies = [
  "chrono",
  "lazy_static",
  "matchers",
- "parking_lot 0.11.0",
+ "parking_lot 0.9.0",
```
Comment (Mark-Simulacrum) [inline Cargo.lock:5349]: Hm, this seems surprising - maybe a mistake?

Comment (calebcartwright) [reply Cargo.lock:5349]: I thought so as well, but it wasn't a change I made by hand (came from x.py check)

Comment (Mark-Simulacrum) [reply Cargo.lock:5349]: hm, can you try reverting it by hand perhaps? if we can't get cargo to cooperate I'd like to page in some cargo folks before landing this

Comment (calebcartwright) [reply Cargo.lock]: done, :crossed_fingers:

**Comment by calebcartwright** [comment]
Also, looks like the bug from https://github.com/rust-lang/rust/pull/80843#discussion_r554476529 is on beta too so would like to nominate for backport though note that'd require backporting the commits from #81027 as well since we had to bump the rustc-ap crates

**Pull request merged**